index.html

```diff
@@ -108,111 +108,124 @@ <h2 id="project_tagline">eiffel-remrem-generate</h2>
 
 <strong><p>Templates for eiffel events are available <a href="http://ericsson.github.io/eiffel-remrem-semantics/">here.</a></p></strong>
 
 
 <p>The message must exist in a json file, for example <strong><em>eiffelactivityfinished.json</em></strong> might contain the following message:<br></p>
 <pre>
 {
-   "msgParams":{
-      "meta":{
-         "type":"eiffelactivityfinished",
-         "domainId":"example.domain",
-         "tags":[
-            "tag1",
-            "tag2"
-         ],
-         "source":{
-            "host":"host",
-            "name":"name",
-            "uri":"http://java.sun.com/j2se/1.3/",
-            "serializer":{
-               "groupId":"G",
-               "artifactId":"A",
-               "version":"V"
-            }
-         }
-      }
-   },
-   "eventParams":{
-      "data":{
-         "outcome":{
-            "conclusion":"TIMED_OUT",
-            "description":"Compilation timed out."
-         },
-         "persistentLogs":[
-            {
-               "name":"firstLog",
-               "uri":"http://myHost.com/firstLog"
-            },
-            {
-               "name":"otherLog",
-               "uri":"isbn:0-486-27557-4"
-            }
-         ]
-      },
-      "links": [{
-         "type": "ACTIVITY_EXECUTION",
-         "target": "aaaaaaaa-bbbb-5ccc-8ddd-eeeeeeeeeee1"
-       }]
-   }
-}</pre>
-
-<h3>
-<a id="with-input-from-the-example-file-shown-above-and-output-to-a-file" class="anchor" href="#with-input-from-the-example-file-shown-above-and-output-to-a-file" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>With input from the example file shown above and output to a file:</h3>
-
-<pre><code> $ java -jar generate-cli.jar -f eiffelactivityfinished.json -t eiffelactivityfinished -r output.json
-</code></pre>
-
-<p>Gives:</p>
-
-<pre>$cat output.json
-[
-  {
-    "data": {
-      "outcome": {
-        "conclusion": "TIMED_OUT",
-        "description": "Compilation timed out."
-      },
-      "persistentLogs": [
-        {
-          "name": "firstLog",
-          "uri": "http://myHost.com/firstLog"
-        },
-        {
-          "name": "otherLog",
-          "uri": "isbn:0-486-27557-4"
-        }
-      ]
-    },
-    "links": [{
-         "type": "ACTIVITY_EXECUTION",
-         "target": "aaaaaaaa-bbbb-5ccc-8ddd-eeeeeeeeeee1"
-    }],
+  "msgParams": {
     "meta": {
-      "domainId": "example.domain",
-      "id": "2075b7d1-83a6-47c8-af1d-1d0f451b1f81",
-      "type": "eiffelactivityfinished",
-      "version": "0.1.0",
-      "time": 1473767230905,
+    "type": "EiffelActivityFinishedEvent",
       "tags": [
         "tag1",
         "tag2"
       ],
       "source": {
+        "domainId": "example.domain",
         "host": "host",
         "name": "name",
-        "uri": "http://java.sun.com/j2se/1.3/",
+        "uri": "http:\/\/java.sun.com\/j2se\/1.3\/",
         "serializer": {
```

Comment:
remove serializer from input and output in serializer try to add meaning full names